### ISSUE TITLE
Scrollbar changes suggestion 

### ISSUE THREAD

**Issue by karsumit94** [Dogfalo/materialize#5891]
Can we have this code added to the main code. 
The UI looks very modern but some browsers scroll bar has a legacy view 

```
/*********** CSS for scrollbar ***********/
::-webkit-scrollbar {
    width: 10px;
}
::-webkit-scrollbar-track {
    background: #f1f1f1; 
}
::-webkit-scrollbar-thumb {
    background: #888; 
}
::-webkit-scrollbar-thumb:hover {
    background: #555; 
}
```

**Comment by MicheleLucini** [comment]
google inbox uses the standard scrollbar. I think that the css for scrollbar should not be included.

**Comment by ray007** [comment]
I'd agree if it wasn't webkit only.

**Comment by karsumit94** [comment]
@MicheleLucini, on the contrary, I found google also using this. Check the screenshot [https://inbox.google.com/]
![image](https://user-images.githubusercontent.com/7498663/39747649-7cd04754-52cb-11e8-8b44-d2877f823e11.png)

**Comment by MicheleLucini** [comment]
True! Didn't notice the sidenav scroll, I was focused on just the body scroll.
I was thinking that the sidenav css is something easy to implement and harder to edit in case of a default materialize version. But maybe it's better to have a default material look handled by them...
